app/src/main/java/org/dhis2/data/forms/dataentry/tablefields/edittext/EditTextCellCustomHolder.java

```diff
@@ -71,6 +71,10 @@
                         )
                 );
             }
+
+            if (hasFocus && itemView.getLeft() > 5) {
+                tableView.scrollToColumnPosition(getAdapterPosition(),3);
```

Comment:
what does 3 represent?

Reply:
It is the default cell cell offset. I'll add a constant.